stores/history.ts

```diff
@@ -0,0 +1,56 @@
+import { atom } from 'nanostores'
+
+import { debounce } from '../lib/time.js'
+import { current, type LchValue } from './current.js'
+
+let $undos = atom<LchValue[]>([])
+let $redos = atom<LchValue[]>([])
+let $prev = atom<LchValue>({ a: 100, c: 0, h: 0, l: 0 })
```

Comment:
It is better to put the current color (we have some color from the beggining).

```suggestion
let $prev = atom<LchValue>(current.get())
```

Reply:
Why do we need `$prev` and can’t use last from `$undos`?

Reply:
it's actually good idea! I removed $prev and code is cleaner, but maybe not that obvious